exasol_script_languages_container_ci_setup/lib/run_start_release_build.py

```diff
@@ -0,0 +1,95 @@
+import logging
+import os
+import re
+from typing import Tuple, Dict
+
+from exasol_script_languages_container_ci_setup.lib.aws_access import AwsAccess
+from exasol_script_languages_container_ci_setup.lib.github_release_creator import GithubReleaseCreator
+from exasol_script_languages_container_ci_setup.lib.release_build import release_stack_name
+
+
+def get_environment_variable_override(env_variable: Tuple[str, str]) -> Dict[str, str]:
+    return {"name": env_variable[0], "value": env_variable[1], "type": "PLAINTEXT"}
+
+
+def _parse_upload_url(upload_url: str) -> int:
+    """
+    upload_url is expected to have the following format: `https://uploads.github.com/repos/exasol/script-languages-repo/releases/123/assets{?name,label}`
+    where `exasol/script-languages-repo` is the repository for which the release will be created and 123 is the id of the release.
+    This method return release id as integer.
+    """
+    res = re.search(r"^https://uploads.github.com/repos/([a-zA-Z0-9\-_/]+)/releases/([\d]+)/assets", upload_url)
+    if res is None:
+        raise ValueError("Parameter upload_url is in unexpected format.")
+    return int(res.groups()[1])
+
+
+def run_start_release_build(aws_access: AwsAccess, project: str, upload_url: str, branch: str) -> None:
+    """
+    This command executes
+    1. Retrieve resources for the release codebuild stack for that given project
+    2. Find the resource with type CodeBuild
+    3. Creates the environment variables override
+    4. Start and wait for batch build
+    :raises:
+        `RuntimeError` if build goes wrong or if anything on AWS CodeBuild is not as expected
+        `ValueError` if project is not found on AWS CodeBuild or if the upload is not in expected format.
+    The upload url is only be used to get the release id.
+    """
+    logging.info(f"run_start_release_build for aws profile {aws_access.aws_profile} for project {project} "
```

Comment:
Might look strange when AWS profile is None

Reply:
Fixed